src/glass/src/app/shared/components/datatable/datatable.component.ts

```diff
@@ -85,6 +91,16 @@ export class DatatableComponent implements OnInit, OnDestroy {
   @Input()
   autoReload: number | boolean = 15000;
 
+  // Row property used as unique identifier for the shown data. Only used if
+  // the row selection is enabled. Will throw an error if property not found
+  // in given columns. Defaults to 'id'.
+  @Input()
+  identifier = 'id';
+
+  // Defines the row selection type. Defaults to no row selection.
+  @Input()
+  selectionType: DatatableSelectionType = DatatableSelectionType.none;
```

Comment:
Using an enum as input property is not so good IMO because in the HTML code you can not use it. With the current implementation it has to look like:
```
      <glass-datatable [data]="devices"
                       sortHeader="path"
                       selectionType="1"
                       [columns]="devicesColumns">
      </glass-datatable>
```
Better would be
```
      <glass-datatable [data]="devices"
                       sortHeader="path"
                       selectionType="single"
                       [columns]="devicesColumns">
      </glass-datatable>
```

```suggestion
  selectionType: 'none' | 'single' | 'multi' = 'none';
```

Reply:
Yes, if you're going to use it in the HTML file, you're right. But there is also the possibility to define variable in the *.ts file, e.g.:

```selectionType: DatatableSelectionType = DatatableSelectionType.single; ```

and then add it to the HTML file:

```[selectionType]="selectionType"```

But well... for me both options are fine.

Reply:
> Yes, if you're going to use it in the HTML file, you're right. But there is also the possibility to define variable in the *.ts file

Right, but IMO this is unnecessary code because this type of configuration is fixed in nearly 99.9%, thus setting the attribute in the HTML code is the way to go.

> Btw: we do have the same for sortDirection:

Hmmm, right. IMO we should also cleanup this issue. It's IMO not the way a developer is using this component and attribute.